expertsystem/particle_list.yml

```diff
@@ -2,16 +2,13 @@
 ParticleList:
   gamma:
     PID: 22
-    Mass: { Value: 0.0 }
+    Mass: 0
     QuantumNumbers:
       Spin: 1
       Charge: 0
       Parity: -1
       Cparity: -1
-      IsoSpin: { Value: 0, Projection: 0 }
-      BaryonNumber: 0
-      Charm: 0
-      Strangeness: 0
+      IsoSpin: 0
```

Comment:
Is the code able to set the projection to zero in this case as well?

Reply:
Yep, there's a check on this. In fact, it will even complain if:
* `abs(projection) > magnitude`
* `magnitude` > 0 and no projection is provided